engine/src/main/java/io/camunda/zeebe/engine/state/migration/DbMigratorImpl.java

```diff
@@ -79,9 +80,14 @@ public void runMigrations() {
         executedMigrations.add(migration);
       }
     }
+    markMigrationsAsCompleted();
```

Comment:
Checking of the version is done in a later stage right?

Reply:
Yep :+1: